_rules/audio-transcript-2eb176.md

```diff
@@ -24,25 +31,27 @@ acknowledgments:
 
 This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- [autoplaying](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#autoplay); or,
```

Comment:
We normally avoid quoting from non-W3C sites, especially vendor ones. If this is just about the `autoplay` attribute like the "3 seconds" rule:
```suggestion
- the element has an `autoplay` [attribute value][] of `true`; or
```